app/src/main/java/no/usn/mob3000/ui/components/settings/SettingsSectionApplication.kt

```diff
@@ -123,14 +126,14 @@ fun SettingsSectionApplication(
             onDismissRequest = { themeExpanded = false }
         ) {
             DropdownMenuItem(
-                text = { Text(stringResource(R.string.settings_section_app_theme_light)) },
+                text = { Text(stringResource(R.string.settings_section_app_theme_light), color = MaterialTheme.colorScheme.onBackground) },
```

Comment:
Same as [with this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847357323). Since there are multiple parameters, it should be indented.